### ISSUE TITLE
openjdk8 x32 build are failing.

### ISSUE THREAD

**Issue by cruzy** [AdoptOpenJDK/openjdk-build#384]
https://ci.adoptopenjdk.net/job/openjdk8_build_x86-32_windows/ - these builds are failing for a month.

Also, these builds are not accessible on your site.  Please, do not drop x32 support, it is still required.

**Comment by karianna** [comment]
@smlambert I'm sure I saw you producing win32 builds in a recent bunch of updates?

**Comment by smlambert** [comment]
I was actually focussed on the openjdk8-openj9 builds for win32 (as part of the openj9 0.9 release), which are building, being tested and published to the website.  I haven't looked at why the openjdk8 (with hotspot) are not compiling.  I can see if I notice anything obvious, but I think it was @sxa555 that enabled those builds originally.

**Comment by cruzy** [comment]
@smlambert how did you publish win32 builds? Could you share the link please?

I can't find any win32 release. There is no link to download win32 binaries on this page: https://adoptopenjdk.net/releases.html?variant=openjdk8

**Comment by gdams** [comment]
@cruzy https://adoptopenjdk.net/nightly.html?variant=openjdk8-openj9

**Comment by cruzy** [comment]
@gdams many thanks for this info. 

Have you declined openjdk8 **hotspot** support for 32-bit platforms?

**Comment by karianna** [comment]
Nope, but it's not our biggest priority, please do join our community at adoptopenjdk.net/slack if you can help!

**Comment by sxa555** [comment]
@cruzy I created that job in jenkins at a similar time to creating the openj9 one. We have so far never had fully working builds with hotspot. As @karianna says we have had specific requests from openj9 to have win32 builds but no-one's so far shouted for hotspot ones, although I'd like us to have parity across the VMs.

Out of interest, what's your need for 32-bit hotspot - is it some specific hardware you're using?

**Comment by sxa555** [comment]
I echo @smlambert's comment but for the x64 JRE - unless you're using 32-bit native JNI code with it the 64-bit one should work ok and if there's any issue please let us know.

**Comment by sxa555** [comment]
(FYI I've tried another HotSpot win32 build run and it's failed in a different place from what it did previously so will still need some investigation and debugging to get that through)

**Comment by szdorant** [comment]
Hi, we are considering AdoptOpenJDK as an alternative JVM due to LTS policy changes by Oracle. Unfortunately some of our clients are still stuck on Win32, and we would like to limit the number of JVMs we test on and support, hence we would go only for Hotspot for now (which also seems to be a safer transition). 64 bit is excellent so far. For us a Win32 Java 8 Hotspot release in time before 2019 January (end of updates from Oracle) seems to be the missing piece for a complete solution.
I'm just writing this down, because I guess this scenario is probably not unique to us.
Do you guys think such a release may be realistic in a few months ?

**Comment by karianna** [comment]
We've asked the Main Adopt community directly, as of yet no volunteers.  I've cast the net out further to the Java User Groups and Java Champions lists.

**Comment by Capa83** [comment]
Hi, we have the question if the JavaFX components will be part of this distribution as well ? Currently in the nightly we don`t see it.

**Comment by SzentnerTsIT** [comment]
Ok, will do!
